app/library/postgres.py

```diff
@@ -74,10 +75,13 @@ async def get_asset_from_postgres(asset_id: str):
 
 
 async def delete_asset_from_postgres(asset_id: str):
-    # TODO: delete from postgres
+    did_delete = delete_asset(asset_id)
 
-    sample_response = {"message": "deleted successfully"}
-    return sample_response
+    if did_delete:
+        response = {"message": "deleted successfully"}
+    else:
+        response = {"message": "asset id " + asset_id + " not found"}
+    return response
```

Comment:
This is just an example of how I envision the deletion being used, but this is for @jaydhulia and @vivianliu0 to implement and change as needed.